src/main/java/org/cactoos/text/Random.java

```diff
@@ -38,7 +37,7 @@
  *
  * @since 0.32
  */
-public final class RandomText implements Text {
+public final class Random implements Text {
```

Comment:
@Iprogrammerr let's name this one `Randomized` (let's not forget to rename test file as well)